packages/devextreme/js/__internal/grids/new/grid_core/editing/popup/component.tsx

```diff
@@ -0,0 +1,70 @@
+import type dxForm from '@js/ui/form';
+import type { RefObject } from 'inferno';
+import { Component } from 'inferno';
+
+import type { DataObject } from '../../data_controller/types';
+import type { FormProperties } from '../../inferno_wrappers/form';
+import { Form } from '../../inferno_wrappers/form';
+import { Popup } from '../../inferno_wrappers/popup';
+import { Scrollable } from '../../inferno_wrappers/scrollable';
+
+export interface Properties {
```

Comment:
Let's rename this interface:

```suggestion
export interface EditPopupProps {
```

Reply:
I'd stick that we shouldn't add prefixes to entities inside file